package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-virtualized-auto-sizer",
-  "version": "1.0.19",
+  "version": "1.1.19",
```

Comment:
Can you revert this bit?

This change could be argued a breaking change (which would require a 2.0 release). Regardless though, I'd prefer to decouple the change itself from the version incrementing (e.g. other changes may also be part of a new version)